spec/03-plugins/38-ai-proxy/09-streaming_integration_spec.lua

```diff
@@ -53,18 +53,18 @@ for _, strategy in helpers.all_strategies() do if strategy ~= "cassandra" then
                 [5] = 'data: {    "choices": [        {            "delta": {},            "finish_reason": "stop",            "index": 0,            "logprobs": null        }    ],    "created": 1712538905,    "id": "chatcmpl-9BXtBvU8Tsw1U7CarzV71vQEjvYwq",    "model": "gpt-4-0613",    "object": "chat.completion.chunk",    "system_fingerprint": null}',
                 [6] = 'data: [DONE]',
               }
-
+              
               local fmt = string.format
               local pl_file = require "pl.file"
               local json = require("cjson.safe")
-
+              
```

Comment:
You may want to configure your editor to automatically strip whitespace off lends of lines.